sdks/sdk-core/src/addresses.ts

```diff
@@ -303,8 +303,8 @@ const UNICHAIN_ADDRESSES: ChainAddresses = {
   tickLensAddress: '0xd5d76fa166ab8d8ad4c9f61aaa81457b66cbe443',
   swapRouter02Address: '0x73855d06de49d0fe4a9c42636ba96c62da12ff9c',
 }
-
-export const CHAIN_TO_ADDRESSES_MAP: Record<SupportedChainsType, ChainAddresses> = {
+// exclude monad because it doesn't have v3 support
```

Comment:
this might be a blocker for routing. do we need why no v3 support? is it because of no v3 protocol deploy on monad testnet?

Reply:
they only want v2 for their competition: https://www.notion.so/uniswaplabs/Monad-Testnet-12fc52b2548b802b8012ff65a9b376ba?pvs=4#12fc52b2548b8071a11ae1c56ae73dcd

Reply:
😎world